### ISSUE TITLE
a11y: add aria-valuetext to slider

### ISSUE THREAD

**Issue by Etat-Plateforme** [react-bootstrap/react-bootstrap#1192]
Feature:
According to http://www.w3.org/TR/wai-aria-practices/#slider, an aria-valuetext property should be set on the slider, containing a label for the current progress:

Example:
`<div class="progress-bar" role="progressbar" aria-valuenow="0" aria-valuemin="0" aria-valuemax="100" aria-valuetext="0%"></div>`

**Comment by jquense** [comment]
The progress bar isn't a slide its `role='progressbar'` which __may__  provide valuetext but it is not required.

http://www.w3.org/TR/wai-aria/roles#progressbar
